companies/smartwool.md

```diff
@@ -0,0 +1,71 @@
+---
+fortune_rank: 233
```

Comment:
![image](https://user-images.githubusercontent.com/21069635/101851497-4a36d880-3b21-11eb-8fc4-828dcd508069.png)

I don't think Smartwool is in the [Fortune500](https://fortune.com/fortune500/2020/search/), you can take this out!

Reply:
Wait, my bad! Now I understand they are owned by VF Corporation. Great find, @Nyette 🤯

Reply:
Thanks! 😄